ios/FluentUI/Pill Button Bar/PillButtonBar.swift

```diff
@@ -542,12 +541,14 @@ open class PillButtonBar: UIScrollView {
     }
 
     private func updateHeightConstraint() {
-        var maxHeight = Constants.minHeight
-        buttons.forEach { maxHeight = max(maxHeight, $0.frame.size.height) }
+        var maxHeight = pillButtonBarTokens.minHeight
+        buttons.forEach { maxHeight = max(maxHeight!, $0.frame.size.height) }
         if let heightConstraint = heightConstraint, maxHeight != heightConstraint.constant {
-            heightConstraint.constant = maxHeight
+            heightConstraint.constant = maxHeight!
```

Comment:
Let's not use explicitly unwrap (`maxHeight!` operator).
You can either do guard let or provide a fallback (default) value if the value is not available from the tokens object.